bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +41,74 @@
+  </p>
+  <img src="{{ static('img/firefox/tiles/suggested-sites.png') }}" alt="{{ _('Screenshot') }}">
 </section>
 
-<section>
-  <h3 data-accordion-role="tab">{{ _('What data is collected when I opt out?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% if LANG in ['en-US'] %}
-        No data is collected when a user deactivates the Enhanced Tiles experience.
-      {% else %}
-      {% trans %}
-        No data is collected when a user chooses to opt out of the enhanced Tiles experience.
-      {% endtrans %}
-      {% endif %}
-    </p>
-  </div>
+<section class="tls-started">
+  <h2>{{ _('If you’re getting started') }}</h2>
+  <p>
+    {{ _('If you’re using Firefox for the first time, the New Tab page is already populated with useful and interesting sites and content.') }} 
+    {{ _('This includes both referrals to sponsored sites from commercial partners, labeled as "Sponsored," and content we consider important to our mission of advancing the public benefit of the Internet.') }}
+  </p>
+  <img src="{{ static('img/firefox/tiles/discover-sites.png') }}" alt="{{ _('Screenshot') }}">
 </section>
 
-<section>
-  <h3 data-accordion-role="tab">{{ _('Where can I learn more?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <ul>
-      {% if LANG in ['en-US'] %}
-      <li><a href="https://support.mozilla.org/kb/how-do-tiles-work-firefox">
-        Mozilla Support: How do Tiles work?
-      </a></li>
-      <li><a href="https://blog.mozilla.org/advancingcontent/">
-        Mozilla Content Services: Advancing Content blog
-      </a></li>
-      <li><a href="https://blog.mozilla.org/netpolicy/2014/11/13/designing-tiles-for-trust/">
-        Mozilla Policy: "Designing Tiles for Trust"
-      </a></li>
-      {% else %}
-      <li><a href="https://blog.mozilla.org/futurereleases/2014/09/04/new-tab-page-experiments-in-firefox-beta/">
-        {{ _('New Tab Experiments in Firefox Beta') }}
-      </a></li>
-      <li><a href="https://blog.mozilla.org/advancingcontent/2014/08/21/a-call-for-trust-transparency-and-user-control-in-advertising/">
-        {{ _('A Call for Trust, Transparency and User Control in Advertising') }}
-      </a></li>
-      <li><a href="https://blog.mozilla.org/netpolicy/2014/08/21/trust-should-be-the-currency/">
-        {{ _('Trust should be the currency') }}
-      </a></li>
-      {% endif %}
-    </ul>
-  </div>
+<section class="tls-privacy">
+  <h2>{{ _('Personalization with your privacy in mind') }}</h2>
+  <p>{{ _('Mozilla promotes a transparent exchange between users and content providers to make the Web enjoyable and sustainable.') }}</p>
+  <p>{{ _('If suggestions are enabled, Firefox will make suggestions to you based on categories identified in your browsing history and will let you know every time why are you seeing specific suggestions.') }}</p>
```

Comment:
I'm French, so I might be wrong, but isn't it supposed to be "every time why you are seeing"?